account_credit_control/migrations/0.3/post-migration.py

```diff
@@ -1,7 +1,7 @@
 # -*- encoding: utf-8 -*-
 import logging
```

Comment:
I bet that this migration script is for older versions and this should be removed.